x-pack/plugins/ml/public/embeddables/anomaly_swimlane/anomaly_swimlane_embeddable.tsx

```diff
@@ -41,12 +42,37 @@ export class AnomalySwimlaneEmbeddable extends AnomalyDetectionEmbeddable<
   private reload$ = new Subject<void>();
   public readonly type: string = ANOMALY_SWIMLANE_EMBEDDABLE_TYPE;
 
+  // API
+  public viewBy = new BehaviorSubject<string | undefined>(this.getInput().viewBy);
```

Comment:
nit. If you're initializing these in the constructor via `embeddableInputToSubject` or `embeddableOutputToSubject`, you don't need to initialize them via `new` here. The helpers will handle that.

Reply:
Good spot, leftovers from the original implementation, updated in https://github.com/elastic/kibana/pull/178729/commits/48d9c509eb99d59ee5f78a3d4fb8cd093dbaca7b